### DISCUSSION TITLE
Can nightly packages be added to main repo?

### DISCUSSION THREAD

**Discussion by MrAdityaAlok** [termux/termux-packages#7969] [Q&A] [answered]
I wanted to add neovim-nightly to unstable-packages, but since it has been deleted, can this be added to main repo?

**Comment by ghost** [comment, marked as answer]
Yes, you can. Just append `-nightly` to package name to notify users.

**Reply by MrAdityaAlok** [reply]
Thank you! I will open a PR soon.
